src/main/scala/org/broadinstitute/dsde/firecloud/dataaccess/DsdeHttpDAO.scala

```diff
@@ -0,0 +1,61 @@
+package org.broadinstitute.dsde.firecloud.dataaccess
+
+import akka.actor.ActorSystem
+import akka.http.scaladsl.Http
+import akka.http.scaladsl.model.headers.{Authorization, OAuth2BearerToken}
+import akka.http.scaladsl.model.{HttpHeader, HttpRequest, HttpResponse, ResponseEntity}
+import akka.http.scaladsl.unmarshalling.Unmarshaller
+import akka.stream.Materializer
+import com.typesafe.scalalogging.LazyLogging
+import org.broadinstitute.dsde.firecloud.model.UserInfo
+import org.broadinstitute.dsde.firecloud.utils.{HttpClientUtils, HttpClientUtilsStandard}
+import org.broadinstitute.dsde.rawls.RawlsExceptionWithErrorReport
+
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.Future
+
+/**
+  * Created by dvoet on 9/2/15.
+  */
+trait DsdeHttpDAO extends LazyLogging {
```

Comment:
lifted from rawls. I am second-guessing whether or not we actually need this _and_ `RestJsonClient.scala` in orch. i have been slowly removing usage of this and favoring `RestJsonClient` where possible

Reply:
not to mention dsde no longer exists...

Reply:
it's gone now! replaced all usages of DsdeHttpDAO code with RestJsonClient